config/base.php

```diff
@@ -193,45 +193,67 @@
 
     /*
     |--------------------------------------------------------------------------
-    | Global Promotion Groups
+    | Global Data
```

Comment:
Wasn't this going to be under it's own globaldata config?

edit: nevermind, for some reason, the review area is showing only certain commits